sdk/translation/azure-ai-documenttranslator/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2024-12-06)
 
+Please note, this package has been deprecated and will no longer be maintained after May 2026. We encourage you to
```

Comment:
This is a beta package and will not get any support when marked as deprecated. The source code will be available in the repo as per the deprecation guidelines.
https://eng.ms/docs/products/azure-developer-experience/develop/sdk-deprecation-guidelines